k8s/aws-k8s-steps.adoc

```diff
@@ -0,0 +1,62 @@
+== Kubernetes Deployment in AWS EKS (Elastic Kubernetes Service)
```

Comment:
Wouldn't it make more sense to place these two documentations in the `documentation` folder instead of `k8s`?

Reply:
Yes, right. Done.